### PR TITLE
added changes to edge and prop load

### PR DESCRIPTION
**Pull request by cheese-stands-alone**
Changed some things to speed up loading files into neo4j.
Loading edges is down to 30s seconds from a few days.
Loading properties is down to a few minutes from a few hours.

### CODE REVIEW

File: graphalyzer-server/src/app.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def load_edges(graphlocation: str, graphid: str):
     from py2neo import Graph
     from py2neo.packages.httpstream import http
     # Increase timeout because it can take a really long time.
-    http.socket_timeout = sys.maxsize
+    http.socket_timeout = 9999
```
Comment (ajbowler) [inline graphalyzer-server/src/app.py:39]: Is this timeout time completely arbitrary or should we be using some standard?

Comment (winder487) [inline graphalyzer-server/src/app.py:39]: I would think it'd be okay to keep it as that for now, and tweak it when we get some data on the average, worst case time taken

Comment (cheese-stands-alone) [inline graphalyzer-server/src/app.py:39]: Arbitrary, the default timeout is 30s so pretty much all loads fail with that. There is a potential issue if neo4j goes down after accepting the initial request.

Comment (ajbowler) [inline graphalyzer-server/src/app.py:39]: Which we can't do much about, unless there's a more graceful way of handling that situation, but that's a separate issue.

So all good from me. :+1:

**Comment by mcsgroi** [comment]
Looks good to me +1

**Comment by ajbowler** [comment]
@algmzestrd is this good for you?

**Comment by algmzestrd** [comment]
Could've sworn I +1'd this a while ago.

**Pull request merged**